app/services/users/manage_credits.rb

```diff
@@ -0,0 +1,50 @@
+module Users
+  class ManageCredits
+    def self.call(*args)
```

Comment:
I know you just copied a pattern we use elsewhere, but I'm not a big fan of obscuring the params of the main entry point of a service object. 

```ruby
def self.call(user, user_params)
  new(user, user_params).call
end
```

This also allows you to have different signatures between `.call` and `#initialize`, e.g.

```ruby
def self.call(user, user_params)
  new(
    user,
    user_params.slice(:add_credits, :remove_credits),
    user_params.slice(:organization_id, :dd_org_credits, :remove_org_credits)
  )
end

def initiliaze(user, user_params, org_params)
  ...
end
```